recipes/libcurl/all/conanfile.py

```diff
@@ -337,38 +396,39 @@ def _build_with_cmake(self):
         cmake.build()
 
     def package(self):
-        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        with tools.chdir(self.source_folder):
+            self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
 
-        # Execute install
-        if self.settings.compiler != "Visual Studio":
-            env_run = RunEnvironment(self)
+            # Execute install
+            if self.settings.compiler != "Visual Studio":
+                env_run = RunEnvironment(self)
 
-            with tools.environment_append(env_run.vars):
-                with tools.chdir(self._source_subfolder):
-                    autotools, autotools_vars = self._configure_autotools()
-                    autotools.install(vars=autotools_vars)
-        else:
-            cmake = self._configure_cmake()
-            cmake.install()
+                with tools.environment_append(env_run.vars):
+                    with tools.chdir(self._source_subfolder):
```

Comment:
same here, avoid chdir if possible and use configure_folder instead.

Reply:
as above, this would create 2 behaviours, it is not the same being in the directory, and configuring a different one

Reply:
I removed all chdir from me, but the original ones, this one, not, 
if this shall be removed, this should be part of a new issue